### ISSUE TITLE
Android version v22.6.0 - black screen on one mobile side (v22.5.1 - OK)

### ISSUE THREAD

**Issue by Perlover** [jitsi/jitsi-meet#12486]
### Description:

On October 24, version 22.6.0 appeared on Google Play. Strange problems were noticed on it - if both sides (mobile clients, Android v12) communicate through their server (own hosted server), then one of the parties (random) always sees itself and the interlocutor, and the other side sees only itself (the interlocutor is a black screen, but there is voice). Version 22.5.1 (F-Droid) does not have this defect when both sides use it.

### Steps to reproduce:

1. Install 22.6.0 from Google Play at two mobile clients (Android 12 for example)
2. To use own Jitsi server (config I will add)
3. Create a connection between two devices. If you add a third device - a computer with a browser (Firefox v106 for example), then all the interlocutors are visible on the computer, but the interlocutor from the computer is also often not visible on a random mobile.

### Expected behavior:

All interlocutors, of course, should see and hear each other. I removed the Google Play version and installed 22.5.1 from F-Droid. The problem has disappeared. I have not observed any problems with the Google Play version before, so I believe that changes were made to the latest version (22.6.0) that broke its work.

### Actual behavior:

To see "Description"

### Server information:

- Operating System: Ubuntu 22.04

Packages versions:

```
jitsi-archive-keyring/stable 1.0.1 all
jitsi-meet-prosody/stable,now 1.0.6644-1 all [installed,automatic]
jitsi-meet-tokens/stable 1.0.6644-1 all
jitsi-meet-turnserver/stable,now 1.0.6644-1 all [installed]
jitsi-meet-web-config/stable,now 1.0.6644-1 all [installed,automatic]
jitsi-meet-web/stable,now 1.0.6644-1 all [installed,automatic]
jitsi-meet/stable,now 2.0.7882-1 all [installed]
jitsi-upload-integrations/stable 0.15.15-1 all
jitsi-videobridge2/stable,now 2.2-45-ge8b20f06-1 all [installed,automatic]
jitsi-videobridge/stable 1126-1 amd64
jitsi-videobridge/stable 1126-1 i386
lua-basexx/stable,now 0.4.1-jitsi1 all [installed,automatic]
lua-cjson-dbgsym/stable 2.1.0.10-jitsi1 amd64
lua-cjson-dev/stable 2.1.0.10-jitsi1 amd64
lua-cjson/stable,now 2.1.0.10-jitsi1 amd64 [installed,automatic]
```

### Client information:

- Browser / app version: 22.6.0
- Operating System: Android 12

### Additional information:

Server configs:

```
/* eslint-disable no-unused-vars, no-var */
/*
 * NOTE: If you add a new option please remember to document it here:
 * https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-configuration
 */
var config = {
    hosts: {
	domain: 'video.my-domain.com',
        muc: 'conference.<!--# echo var="subdomain" default="" -->video.my-domain.com'
    },
    bosh: '//video.my-domain.com/http-bind',
    websocket: 'wss://video.my-domain.com/xmpp-websocket',
    testing: {
    },
    flags: {
        sourceNameSignaling: true,
        sendMultipleVideoStreams: true,
        receiveMultipleVideoStreams: true
    },
    enableNoAudioDetection: true,
    disableShowMoreStats: false,
    enableNoisyMicDetection: true,
    channelLastN: -1,
    enableWelcomePage: true,
    defaultLanguage: 'ru',
    p2p: {
        enabled: true,
        stunServers: [
            { urls: 'stun:turn-video.my-domain.com:3478' },
            { urls: 'stun:turn-video.my-domain.com:443' }
        ]
    },
    analytics: {
    },
    deploymentInfo: {
    },
    mouseMoveCallbackInterval: 1000,
    makeJsonParserHappy: 'even if last key had a trailing comma'
};
/* eslint-enable no-unused-vars, no-var */

```

**Comment by saghul** [comment]
Thanks for the report! Are you able to grab a logcat of the device with version 22.6.0? Also, can you reproduce this when connecting to meet.jit.si ?

**Comment by erengms** [comment]
same issue, one of the phones stays on black screen

**Comment by saghul** [comment]
> same issue, one of the phones stays on black screen

Can you please answer the questions from my previous comment?

**Comment by Perlover** [comment]
> Are you able to grab a logcat of the device with version 22.6.0?

How can I make it? Please any doc for this and I will try to do it

> Also, can you reproduce this when connecting to meet.jit.si ?

I will try and will write to you

I noticed one feature. I just communicated through my Jitsi service (described in my first answer), I had version 22.5.1 (F-Droid source), and the interlocutor had version 22.6.0 (Google Play) and both saw each other. I have an assumption that the problem appears only when both mobile clients have version 22.6.0

> Can you please answer the questions from my previous comment?

You quoted an answer from another user, but you kind of answered me. I need to understand - how do I collect logcat logs for you :)

**Comment by Perlover** [comment]
New information. I used my server and my two Android devices again. I put 22.6.0 on one (Google Play), 22.5.1 (F-Droid) on the other. Both devices saw each other and everything was fine. Then I put version 22.6.0 on the second one (Google Play version). The problem reappeared - a dark screen of the remote interlocutor was observed on one of the devices. In several tests, I found that a dark screen appears on the one who first entered the video conference. I'll throw you a link in a private message to my personal video server. You can try it yourself. In addition, you can collect logcat logs yourself, if you have an Android device and save me from this procedure that is still incomprehensible to me.

P.S. Now I will try your server (meet.jit.si) and I will write the result in a new message.

**Comment by Perlover** [comment]
I just tested your server: https://meet.jit.si/OkYardsStageAll
The same problem!

**Comment by saghul** [comment]
I could reproduce the issue!

Android <-> Android fails (1 participant gets a black screen)
Android <-> iOS works

If you disable P2P it will work (this can be done in settings, before you join the meeting).

Not sure what's up there, since the iOS and Android implementation are pretty much the same.

Since one of the next changes is the migration to unified plan, I'm not sure we'll have the time to dig into it since unified plan introduces a lot of changes in that area...

**Comment by saghul** [comment]
This was fixed in 23.2.0.
